models/samplers/pcd.py

```diff
@@ -9,21 +9,21 @@
 
 class PCD(BaseSampler):
     
-    def __init__(self, batchSize, RBM, **kwargs):
+    def __init__(self, batch_size, RBM, **kwargs):
```

Comment:
I renamed the `batchSize` variable to `batch_size` to adhere to the convention used in the rest of the code:

- class name start with capital letter and do not use underscores
- methods start with lower case letter and do not use underscores
- variables start with lower case letter and use underscores